reverse_engineering/helpers/snowflakeHelper.js

```diff
@@ -422,11 +423,12 @@ const getConnectionTimeoutError = timeout => {
 	return error;
 };
 
-const getAccount = hostUrl =>
-	(hostUrl || '')
-		.trim()
-		.replace(/\.snowflakecomputing\.com.*$/gi, '')
-		.replace(/^http(s)?:\/\//gi, '');
+const getAccount = hostUrl => {
```

Comment:
@WilhelmWesser I'm wondering if this is actually really needed to escape anything when there is a snowflake dns.  It almost impossible in such a case to actually enter an ipv6 (or even ipv4) address to target this cloud saas platform.

Reply:
You are right. I doubt that users will seek the ipv6 instead of the resolved name from Snowflake console. I just added it "in case"

Reply:
I will close the PR